genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaJobSearchServiceImpl.java

```diff
@@ -362,4 +363,43 @@ public String getJobHost(@NotBlank final String jobId) throws GenieException {
             throw new GenieNotFoundException("No job execution found for id " + jobId);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Long getActiveJobCountByUserName(@NotBlank final String userName) {
+        log.debug("Called with username {}", userName);
+
+        if (StringUtils.isBlank(userName)) {
```

Comment:
You shouldn't have to do this as you already have `@NotBlank` on the method parameter. Additionally what does `String.valueOf("    ")` or other blank get you